client/package.json

```diff
@@ -82,5 +90,25 @@
     "plugins": [
       "@babel/transform-runtime"
     ]
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "jsdom",
```

Comment:
Do we really need to use jsdom? By default there is a way to properly render all the components and event emulate events on them :/

Reply:
Yes we need, because at least we use d3, which relies on DOM

Reply:
Yes, thats right